### PR TITLE
Cypress test. Cloud Storage page.

### PR DESCRIPTION
**Pull request by dvkruchinin**
### Motivation and context
Added test to check "Cloud Storage" page and check select files from "Cloud Storage" when creating a task.
It is difficult to fully implement the functionality check because access to cloud storage is required.

### Checklist

- [x] I submit my changes into the `develop` branch
- [ ] I have added description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [x] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2021 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

File: tests/cypress/integration/actions_tasks3/case_105_cloud_storage.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+                cloudStorageFormElements.forEach(($el) => {
+                    cy.get($el).should('exist');
+                });
+            });
+            cy.get('#provider_type').click();
+            cy.contains('.cvat-cloud-storage-select-provider', 'AWS').click();
+            cy.get('.cvat-cloud-storage-form').within(() => {
+                cy.get('#resource').should('exist');
+                cy.get('#credentials_type').should('exist');
+                cy.get('#region').should('exist');
```
Comment (Marishka17) [inline tests/cypress/integration/actions_tasks3/case_105_cloud_storage.js:39]: Could we also select some from existing regions, try to add another region, try to press button `Add region` when `key`,  `name` fields are incorrect (e.g are empty) and check that notification(`cvat-incorrect-add-region-notification`) will appear?

**Review by Marishka17 — COMMENTED** [review]
Could we also try to add, remove some manifests in the cloud storage creating form?

**Comment by dvkruchinin** [comment]
> Could we also try to add, remove some manifests in the cloud storage creating form?

Steps added.

File: tests/cypress/integration/actions_tasks3/case_105_cloud_storage.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+            });
+            cy.contains('.cvat-cloud-storage-select-provider', 'AWS').click();
+            cy.contains('.cvat-cloud-storage-select-provider', 'Azure').click();
+            cy.get('.cvat-cloud-storage-form').within(() => {
+                cy.get('#resource').should('exist');
+                cy.get('#credentials_type').should('exist');
```
[the remark this answers is not in the document]
Comment (dvkruchinin) [reply tests/cypress/integration/actions_tasks3/case_105_cloud_storage.js]: Added

File: tests/cypress/integration/actions_tasks3/case_105_cloud_storage.js (hunk 1 above)
Comment (dvkruchinin) [reply tests/cypress/integration/actions_tasks3/case_105_cloud_storage.js]: Added

File: tests/cypress/integration/actions_tasks3/case_105_cloud_storage.js [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+                    cy.contains('button', 'Add region').click()
+                });
+            cy.get('.cvat-incorrect-add-region-notification').should('exist');
+            cy.closeNotification('.cvat-incorrect-add-region-notification');
+        });
+
+        it('Check "Azure Blob Container" provider fields.', () => {
+            cy.contains('.cvat-cloud-storage-select-provider', 'AWS').click();
+            cy.contains('.cvat-cloud-storage-select-provider', 'Azure').click();
+            // Check fields with "Key id and secret access key pair"
```
Comment (Marishka17) [inline tests/cypress/integration/actions_tasks3/case_105_cloud_storage.js:94]:
```suggestion
            // Check fields with "Account name and SAS token"
```

**Review by Marishka17 — APPROVED** [review]

File: tests/cypress/integration/actions_tasks3/case_105_cloud_storage.js (hunk 3 above)
Comment (dvkruchinin) [reply tests/cypress/integration/actions_tasks3/case_105_cloud_storage.js]: Fixed it locally

**Review by Marishka17 — DISMISSED** [review]

**Pull request merged**